jboss-kie-smartrouter/added/launch/jboss-kie-smartrouter.sh

```diff
@@ -47,6 +48,18 @@ function configure_mem_ratio() {
     export JAVA_INITIAL_MEM_RATIO=${JAVA_INITIAL_MEM_RATIO:-25}
 }
 
+function configure_logger_config_file() {
+  # use a default level if the env var is unset
+	local LOG_LEVEL=${LOG_LEVEL:-INFO}
+  sed -i "s/{LOG_LEVEL}/${LOG_LEVEL}/" ${CONFIG_DIR}/logging.properties
+  # write all packages with loglevel
+
+  local PACKAGES_LOG_LEVEL=$(echo $PACKAGES_LOG_LEVEL_CONF | awk '{gsub(",", "\\n")}1')
```

Comment:
you also need to make sure to list all supported log levels, use the JUL pattern.
here is an example:
https://github.com/wildfly/wildfly-cekit-modules/blob/master/jboss/container/wildfly/launch/logger-category/added/launch/configure_logger_category.sh